sdk/core/azure-core-amqp/pom.xml

```diff
@@ -8,7 +8,7 @@
     <groupId>com.azure</groupId>
     <artifactId>azure-client-sdk-parent</artifactId>
     <version>1.7.0</version> <!-- {x-version-update;com.azure:azure-client-sdk-parent;current} -->
-    <relativePath>../../../pom.client.xml</relativePath>
+    <relativePath>../../parents/azure-sdk-client-parent</relativePath>
```

Comment:
No I don't think so. Whenever you specify a path Maven will default try to find the ```pom.xml``` file in that directory, and since we are having one directory per parent POM I think this will fork fine. I did notice that I had my POM name incorrect though (should be ```azure-client-sdk-parent``` instead of ```azure-sdk-client-parent``` :0)